controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -228,9 +265,14 @@ def _prepare_lsvdisk_to_return_mapless_target_volume(self):
     def _prepare_lsvdisk_to_return_none(self):
         self.svc.client.svcinfo.lsvdisk.return_value = self._mock_cli_object(None)
 
-    def _prepare_mocks_for_get_snapshot(self):
+    def _prepare_mocks_for_delete_snapshot(self):
         target_cli_vol = self._get_mapped_target_cli_vol()
         self.svc.client.svcinfo.lsvdisk.return_value = self._mock_cli_object(target_cli_vol)
+        target_cli_vol.FC_id = 'many'
```

Comment:
wouldn't it be more appropriate to move this line one line up?

Reply:
done